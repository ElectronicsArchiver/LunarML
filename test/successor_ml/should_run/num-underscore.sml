print (Int.toString 123_45 ^ "\n");
print (Int.toString 123___45 ^ "\n");
print (Int.toString 0_123___45 ^ "\n");
print (Int.fmt StringCvt.HEX 0x1_2_3_4_5 ^ "\n");
print (Bool.toString (123___4.5_6 > 1_2_3_4.5) ^ "\n");
print (Bool.toString (1e2_3 > 9.9_9_9e2_2) ^ "\n");
