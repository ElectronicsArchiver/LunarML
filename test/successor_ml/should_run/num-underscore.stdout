12345
12345
12345
12345
true
true
